CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 ### New features
 
+#### Check checkboxes by using the `checkedValues` option
+
+When using the Checkboxes nunjucks macro you can now use `checkedValues` option to indicate which checkboxes should be checked when the page loads.
+
+For example, this would check the checkboxes that have the values 'foo' and 'bar':
+
+```
+checkedValues: ['foo', 'bar']
+```
+
+This is an alternative to setting the `checked` option on each individual checkbox item.
+
+This change was introduced in [pull request #2616: Allow selecting options by passing current values](https://github.com/alphagov/govuk-frontend/pull/2616).
+
+#### Check a radio button by using the `checkedValue` option
+
+When using the Radios nunjucks macro you can now use `checkedValue` option to indicate which radio should be checked when the page loads.
+
+For example, this would check the radio that has the values 'foo':
```

Comment:
```suggestion
For example, this code would check the radio that has the value 'foo':
```

Reply:
I've tweaked these slightly to put the code inline, like:

> For example, `value: 'red'` would check the radio that has a `value` of 'red'.